### ISSUE TITLE
TAS Combined Map Checklist

### ISSUE THREAD

**Issue by longlostblake** [7Cav/7thCavalry_Training_Missions#31]
- [x] Redefine hidden marker 'owners'
- [x] Revert back to old unit classnames
- [x] Create alternate version without USAF
- [x] Talk with instructor to remedy Drop 5 (CH47 drop).

**Comment by longlostblake** [comment]
On Hold until mod pack update

**Comment by longlostblake** [comment]
Modpack indicates reverting back to RHS assets. Will need to talk to instructor to see what they want.

**Comment by longlostblake** [comment]
Resuming work on this.

- [] Drop 5 to be replaced with C130 adjustments to DZ needed.
- [] USAF Assets/dependancies need replaceing

**Comment by longlostblake** [comment]
Replaced Drop 5 and USAF Assets. v2_4 released. Waiting on feedback.

**Comment by longlostblake** [comment]
Never received feedback
